index.html

```diff
@@ -4,430 +4,336 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ben Omayio</title>
+    <title>Ben Omayio Portfolio</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
-    <div class="desktop">
-      <main>
-        <header>
-          <a class="logo" href="#">kiboma</a>
-          <ul>
-            <li><a href="#">Portoflio</a> </li>
-            <li><a href="#">About</a> </li>
-            <li><a href="#">Contact</a> </li>
-          </ul>
-        </header>      
-        
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo">KIBS</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#portfolio" class="nav_link">Portfolio </a>
+          </li>
+          <li class="nav_item">
+            <a href="#about" class="nav_link">About </a>
+          </li>
+          <li class="nav_item">
+            <a href="#contact" class="nav_link">Contact</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"> </span>
+          <span class="bar"> </span>
+          <span class="bar"> </span>
```

Comment:
- [ ] Kindly remove the white gap at the top when mobile menu is open.
- [ ] Kindly add some space at the top for the menu close button as in the Figma design
#### Design
![mobile-menu-top-figma](https://user-images.githubusercontent.com/44057863/167738782-4d9fd7bd-7b37-424c-bce6-7ea6664b7d18.png)
#### Impl
![mobile-menu-top-white-strip](https://user-images.githubusercontent.com/44057863/167738688-11aeb22c-1976-44cf-a920-5b8a748d26d5.png)